File: 2608-count-the-digits-that-divide-a-number/2608-count-the-digits-that-divide-a-number.py
class Solution(object):
    def countDigits(self, num):
        """
        :type num: int
        :rtype: int
        """
        
        text = str(num)
        n = 0

        for i in text:
            if((num % (int(i))) == 0):
                n+=1
        return n
    

File: 2608-count-the-digits-that-divide-a-number/2608-count-the-digits-that-divide-a-number.java
class Solution {
    public int countDigits(int num) {

        char[] s1 = ("" + num).toCharArray();
        int n = 0;

        for(int i = 0; i < s1.length; i++){
            if((num % (s1[i]-'0')) == 0){ 
                n++;
            }
        }
        return n;
    }
}